mlrun/utils/helpers.py

```diff
@@ -266,7 +266,7 @@ def normalize_name(name):
         warnings.warn(
             "Names with underscore '_' are about to be deprecated, use dashes '-' instead."
             "Replacing underscores with dashes.",
-            FutureWarning,
+            PendingDeprecationWarning,
```

Comment:
Why did you replace this back to `PendingDeprecationWarning`?

Reply:
it made more sense in the context but the user won't see it so returned to `FutureWarning`